gulpfile.js

```diff
@@ -361,7 +361,7 @@ function buildExamples(watch) {
   // Also update test-example-validation.js
   buildExample('a4a.amp.html');
   buildExample('ads.amp.html');
-  buildExample('ads.with.script.amp.html');
+  buildExample('ads-legacy.amp.html');
```

Comment:
maybe the ads with script should be the canonical? we do want people to explicitly add the script tag.